src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -155,12 +155,11 @@ internal class MemberSignatureComparer : IEqualityComparer<Symbol>
         /// <summary>
         /// This instance is used to check whether one property or event overrides another, according to the C# definition.
         /// <para>NOTE: C# ignores accessor member names.</para>
-        /// <para>CAVEAT: considers return types so that getters and setters will be treated the same.</para>
         /// </summary>
         public static readonly MemberSignatureComparer CSharpAccessorOverrideComparer = new MemberSignatureComparer(
             considerName: false,
             considerExplicitlyImplementedInterfaces: false, //Bug: DevDiv #15775
-            considerReturnType: true,
+            considerReturnType: false,
```

Comment:
I see that this comparer is used for properties/indexers and also events. Can you add tests for event accessors?

Reply:
Sure, but events have an add accessor which will prevent overriding with a different return type.

---
In reply to: [413965442](https://github.com/dotnet/roslyn/pull/43576#discussion_r413965442) [](ancestors = 413965442)